support/ml-advice/index.rst

```diff
@@ -14,8 +14,156 @@
 .. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link and explicit title using the syntax :ref:`link text <label-name>`.
 .. A warning will alert you of identical labels during the linkcheck process.
 
-########################################
-Advice for machine learning integrations
-########################################
+################################################################################
+Considerations for integrating AI with Citizen Science in Rubin
+################################################################################
 
 *Advice, resources, and best practices for the integration of machine learning techniques into Citizen Science projects to be added here.*
+
+
+
+Overview
+========
+
+.. Purpose
+The goal of this page is to provide background, references, and advice for Rubin users who would like to use artificial intelligence (AI) -- otherwise known as machine learning (ML) -- within or alongside their citizen science (CS) research workflows. AI and CS are overlapping and complementary procedures for rapidly assessing (typically, classifying) scientific data. They have similar and unique pros and cons, and there are multiple ways to use these two methods in concert to improve the quality of your scientific results. In addition to this advice, Rubin CS offers tutorials for CS in Rubin and expedited access to the Zooniverse CS platform, and Rubin CST offers tutorials on AI. We recommend studying AI, including performing your own small computational studies, before embarking on a CS project that integrates AI. 
+
+.. Context for the rest
+Almost all of the discussion on this page is in terms of classification tasks; we will use the specific task of classifying 'stars' and 'galaxies' as an example when needed. Additionally, our discussion will focus on classifying images, as this is the most widespread application of both AI and CS. When referring to 
+
+
```

Comment:
Following on the similarities/differences, would it be possible to provide some advice on when to use AI vs CS? Sort of like a little checklist, like "If your project is A, B, C, then CS"?

Reply:
Maybe not here, maybe after the "Comparing AI and CS" section.